pkg/agent/openflow/pipeline.go

```diff
@@ -2299,20 +2301,17 @@ func (f *featureService) serviceLearnFlow(groupID binding.GroupIDType,
 	// the same endpoint because of connection tracking; and that is also the desired behavior.
 	learnFlowBuilderLearnAction := flowBuilder.
 		Action().Learn(SessionAffinityTable.GetID(), priorityNormal, 0, affinityTimeout, cookieID).
```

Comment:
If "isIPv6" used for many APIs with learn action, do you think it is easier if we can add it as a property when constructing the learnAction? Then it could use as a private property of the action

Reply:
Done as you said, and that looks much better.